web/pdf_find_controller.js

```diff
@@ -335,7 +335,12 @@ class PDFFindController {
       this._calculateWordMatch(query, pageIndex, pageContent, entireWord);
     }
 
-    this._updatePage(pageIndex);
+    // In the event that the page *and* its textLayer have both finished
```

Comment:
I think I'm just missing something, but could you explain a bit how this comment relates to the added condition below? In other words, how does the condition below match the fact that both the page and text layer are finished rendering? As far as I can tell we're only checking if the page index matches the one of the currently active match. I think we can tell by the fact that we have an active match that the page and the text layer must be have been rendered already, but is that is also what you meant? If so, we should clarify that a bit in the comment.